src/java/org/apache/cassandra/cql3/UpdateParameters.java

```diff
@@ -274,6 +274,14 @@ public Row buildRow()
         return built;
     }
 
+    @Override
+    public void close()
+    {
+        if (builder == null) return;
+        builder.reset(); // Clear resources
+        builder = null;
```

Comment:
I guess it's nice to have this so we'll explode if someone tries to use a closed `UP` :D